src/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -39,6 +39,8 @@ public static partial class Http
             public static string SelfSignedCertRemoteServer => GetValue("COREFX_HTTPHOST_SELFSIGNEDCERT", "https://self-signed.badssl.com/");
             public static string RevokedCertRemoteServer => GetValue("COREFX_HTTPHOST_REVOKEDCERT", "https://revoked.grc.com/");
 
+            public static string EchoClientCertificateRemoteServer => GetValue("COREFX_HTTPHOST_ECHOCLIENTCERT", "https://corefx-net-tls.azurewebsites.net/EchoClientCertificate.ashx");
```

Comment:
@davidsh, the other website's source code is at https://github.com/dotnet/corefx/tree/master/src/Common/tests/System/Net/Prerequisites 
We should keep everything in one place: please update the source-code in this repo part of this change.